public/views/events.list.html

```diff
@@ -8,7 +8,7 @@
     </div>
   </md-toolbar>
   <md-list flex>
-    <md-virtual-repeat-container id="rhp-virtual-repeat-container">
```

Comment:
Interestingly, this is the only virtual repeat list that doesn't have "slack" at the bottom when all the way scrolled down.